test/images/sample-apiserver/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=k8s.gcr.io/ubuntu-slim:0.12
```

Comment:
Why are we using `k8s.gcr.io/ubuntu-slim:0.12` base image and why not ubuntu from hub.docker.com? I always have this confusion about the version they are using 0.12, which never matches upstream tag.!